training/src/main/java/wolox/training/repositories/ClientRepository.java

```diff
@@ -11,5 +11,5 @@
     Client findByUsername(String username);
 
 
-    List<Client> findByUsernameContainingAndBirthdateBetween(String username, LocalDate from, LocalDate to);
+    List<Client> findByUsernameContainingIgnoreCaseAndBirthdateBetween(String username, LocalDate from, LocalDate to);
```

Comment:
You can put an AllIgnoreCase at the end to say that all string compares should be case insensitive. Just so you know ;)